frontend/herd.tsx

```diff
@@ -0,0 +1,38 @@
+/**
+ * @file This file contains the Herd function. This function fetches herd for a
+ *       given `herdId` (parsed from the url), as well as the individuals
+ *       belonging to that herd.
+ */
+import React from 'react'
+import {Link, useParams} from "react-router-dom";
+
+import { get } from './communication';
+
+/**
+ * Shows herd information, with a list of all individuals belonging to that
+ * herd.
+ */
+export function Herd() {
+  let { herdId } = useParams();
+  const [herd, setHerd] = React.useState(undefined)
```

Comment:
This will give a type error later since it will make he type of `herd` be `undefined`. One can get around this by making type interfaces for what it will return:

```
interface Herd {
   ...
}

... const [herd, setHerd] = React.useState(undefined as undefined | Herd)
```

or just saying that it's any if you prefer it to be untyped:

```
... const [herd, setHerd] = React.useState(undefined as any)
```

Reply:
I like having a defined type, but I'll set it to `any` for this part of development and fix it once we have the first testing feedback from the users... Or is that lazy?